rest-api-spec/src/main/resources/rest-api-spec/test/indices.stats/40_updates_on_refresh.yml

```diff
@@ -0,0 +1,66 @@
+---
+setup:
+
+  - do:
+      indices.create:
+          index:  test1
+          wait_for_active_shards: all
+          body:
+              settings:
+                # Limit the number of shards so that shards are unlikely
+                # to be relocated or being initialized between the test
+                # set up and the test execution
+                index.number_of_shards: 3
+                index.number_of_replicas: 0
+              mappings:
+                  properties:
+                      bar:
+                          type: text
+                          fielddata: true
+                          fields:
+                              completion:
+                                  type: completion
+
+  - do:
+      cluster.health:
+        wait_for_no_relocating_shards: true
```

Comment:
Do we need the `wait_for_events: languid` hack here to guard against a relocation completing but delayed reroute causing a short period before the next relocation happens (in case more are necessary to balance cluster)?

Reply:
... maybe ...

but it doesn't hurt - added in e3262eb8ae5d6e1dd59fcc84cfabafea3be58d06.